courses-app/src/components/Courses/Courses.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react';
+
+import { BUTTONS_TEXT, mockedCoursesList } from '../../constants';
+
+import { Button } from '../../common/Button/Button';
+import CreateCourse from '../CreateCourse/CreateCourse';
+import CourseCard from './components/CourseCard/CourseCard';
+import SearchBar from './components/SearchBar/SearchBar';
+
+import { StyledCourses, StyledHeadingCourses } from './Courses.style';
+import { CourseType } from '../../types';
+
+const Courses = () => {
+	const [newCourse, setNewCourse] = useState(false);
```

Comment:
These names are unclear. Example: `courseFormVisible` is much cleaner

Reply:
Change.